docs/reference/snapshot-restore/register-repository.asciidoc

```diff
@@ -127,9 +130,27 @@ Other repository types are available through official plugins:
 
 * {plugins}/repository-hdfs.html[Hadoop Distributed File System (HDFS)]
 
-You can also use alternative implementations of these repository types, such as
-MinIO, as long as they're compatible. To verify a repository's compatibility,
-see <<snapshots-repository-verification>>.
+You can also use alternative storage implementations with these repository
+types, as long as the alternative implementation is fully compatible. For
```

Comment:
I think "fully" is maybe too much. At least we do not claim that minio is "fully" compatible. I'd say compatible is fine, we do elaborate on it in the following section?

Reply:
I'd rather err on the side of caution here. Everyone calls their implementation "compatible" even if it's only a subset of endpoints, but few are willing to claim _full_ compatibility. Experience shows it's best not to leave room for debate in this area.